src/core/connection.h

```diff
@@ -340,6 +340,12 @@ typedef struct QUIC_CONNECTION {
     //
     QUIC_WORKER* Worker;
 
+    //
+    // The partition this connection is currently assigned to. It is changed at
+    // the same time as the worker.
+    //
+    QUIC_PARTITION* Partition;
```

Comment:
Could this comment clarify that the partition can be set without the worker being set? It gives the impression the two are always set at the same time, while we can have a partition and no worker IIUC.